collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/BikramSambatDatePickerDialog.java

```diff
@@ -49,7 +49,7 @@ public void onResume() {
 
     @Override
     protected void updateDays() {
-        BikramSambatDate bikramSambatDate = new BikramSambatDate(getYear(), Arrays.asList(monthsArray).indexOf(getMonth()), getDay());
+        BikramSambatDate bikramSambatDate = new BikramSambatDate(getYear(), Arrays.asList(monthsArray).indexOf(getMonth()) + 1, getDay());
```

Comment:
A few lines below we use +1 creating BikramSambatDate object in this place we also should I don't know why before it didn't cause any exception but after updating to v1.1.0 it does (for value 0) probably because of the fact that some other changes have been also included in the pr https://github.com/medic/bikram-sambat/pull/10